CHANGES.rst

```diff
@@ -28,6 +28,8 @@ mast
 - Filtering by file extension or by a string column is now case-insensitive in ``MastMissions.filter_products``
   and ``Observations.filter_products``. [#3427]
 
+- Switch to use HTTP continuation for partial downloads. [#3448]
```

Comment:
Not sure this needs a changelog as being an internal change, but probably doesn't hurt either.